operations/libero.md

```diff
@@ -0,0 +1,47 @@
+# Libero howto
+
+The Libero is the person handling 1st level support and on-call work during business hours. The Libero’s main duty is to provide swift help for our customers and us.
+
+This is the description what we expect the Libero to do and how to handle incoming requests.
```

Comment:
There's a subject change mid-sentence. Make it simple: "Here's what we expect from the Libero in detail."